UI/window-basic-filters.cpp

```diff
@@ -1169,18 +1165,16 @@ void OBSBasicFilters::FilterNameEdited(QWidget *editor, QListWidget *list)
 	editActive = false;
 }
 
-void OBSBasicFilters::AsyncFilterNameEdited(
-	QWidget *editor, QAbstractItemDelegate::EndEditHint endHint)
+void OBSBasicFilters::AsyncFilterNameEdited(QWidget *editor,
+					    QAbstractItemDelegate::EndEditHint)
```

Comment:
`QAbstractItemDelegate::EndEditHint` can be removed

Reply:
Doesn't work.

Reply:
Sure works for me.
Can't say why exactly it doesn't work for you without knowing the error, but make sure to also change the arguments used in the `SIGNAL` macro.

Reply:
Fixed